services/ui-src/src/components/cards/EntityCardByStepType.tsx

```diff
@@ -9,12 +9,16 @@ import {
   ReportType,
 } from "types";
 import { useStore } from "utils";
+import { ObjectiveProgressCardBottomSection } from "./ObjectiveProgressCardBottomSection";
 
-export const EntityStepCardTopSection = ({
+export const EntityCardByStepType = ({
```

Comment:
This file seems a bit unruly. I'd prefer each step component to pulled put into its own file since there's no shared HTML between them.